### PR TITLE
Bug 1799054: Add quotations around the cluster-monitoring CSV annotation.

### PR DESCRIPTION
**Pull request by timflannagan1**

### CODE REVIEW

File: charts/metering-ansible-operator/values.yaml [hunk 1]
```diff
@@ -314,8 +314,8 @@ olm:
         version: v1
 
     annotations:
-      operatorframework.io/suggested-namespace: openshift-metering
-      operatorframework.io/cluster-monitoring: true
+      "operatorframework.io/suggested-namespace": openshift-metering
+      "operatorframework.io/cluster-monitoring": "true"
```
Comment (pruan-rht) [inline charts/metering-ansible-operator/values.yaml]: @timflannagan1 I see you quoted the keys here, but not the other places.

Comment (timflannagan1) [reply charts/metering-ansible-operator/values.yaml]: Fixed.

**Pull request merged**